LandSandBoat/server
# [sql] Formatting

**_I affirm:_**

## What does this pull request do?

Format SQL files using dbtool.
Changes `monstrosity_species.look` to VARBINARY so it keeps its hex formatting.

## Steps to test these changes
Visually confirm only formatting changes.
Verify no changes are reported by Git after running the following commands:
```ps
py -3 dbtool.py update full
py -3 dbtool.py dump
```
